src/test/java/com/BrewMES/demo/api/ControllerTest.java

```diff
@@ -0,0 +1,12 @@
+package com.BrewMES.demo.api;
+
+
+import org.junit.jupiter.api.BeforeEach;
+
+public class ControllerTest {
+    @BeforeEach
+    void setup() {
+
+    }
+
+}
```

Comment:
Either implement test completely or remove it.

Reply:
https://github.com/Semester3-03/Java-application/pull/72/commits/47b47a0a132d5f716a43a634ddb0fb813b95a4c9